docs/source/platform/operation-registry.md

```diff
@@ -365,18 +365,24 @@ the new storage location, upgrade:
 - `apollo-server` or `apollo-server-<variant>` to `2.6.3`
 - `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1`.
 
-If operations have not been registered since June 6th, the plugin will
+If operations have not been registered since June 6th, 2019, the plugin will
 fallback on the old location. With `debug` enabled, the plugin will log which
 manifest is used and the operations added.
 
-#### To target variants/tags other than "current"(default, no tag)
+#### Target variants/tags other than `current`(default, no variant/tag)
 
 Operations are now registered against a variant/tag and retrieved from the
-manifest specific to a variant/tag. When a variant/tag is set, the operation
+manifest specific to a variant/tag. This does not change the default behavior
+of the Apollo platform, which universally uses the variant/tag `current` when
+unspecified. Variants/tags represent different environments, such as
+`staging`, `test` or `prod`, so they do not share data. This means that
+operations must be re-registered when using or transitioning to a new
+variant/tag.
+
+When a variant/tag is set in `apollo client:push`, the operation
 will be validated against the latest schema published under that variant/tag
```

Comment:
I thought we took out operation validation, no?

Reply:
We [validate the operation](https://github.com/mdg-private/monorepo/blob/5ed1482a25ba0a57dbc50fab0197c5cea119de4b/mdg/engine/api/gateway/src/operationRegistryService/index.ts#L301-L304) and fail the publish if they don't validate